beltz-padagogik.csl

```diff
@@ -111,7 +126,14 @@
   <macro name="year">
     <group delimiter="/">
       <date date-parts="year" form="text" variable="original-date"/>
-      <date date-parts="year" form="text" variable="issued"/>
+      <choose>
+        <if match="any" is-uncertain-date="issued">
```

Comment:
Should 'no date" also be shown if there is no `issued`?

Reply:
I made a change, but I'm not sure I understand (awkward smile)

Reply:
I meant, I would think the style should also show "no date" if `variable="issued" match="none"`